astra/src/main/java/com/slack/astra/elasticsearchApi/ElasticsearchApiService.java

```diff
@@ -92,24 +92,30 @@ public HttpResponse clusterMetadata() {
   @Post
   @Blocking
   @Path("/_msearch")
-  public HttpResponse multiSearch(String postBody) throws Exception {
-    LOG.debug("Search request: {}", postBody);
+  public HttpResponse multiSearch(String postBody) {
 
-    CurrentTraceContext currentTraceContext = Tracing.current().currentTraceContext();
-    try (var scope = new StructuredTaskScope<EsSearchResponse>()) {
-      List<StructuredTaskScope.Subtask<EsSearchResponse>> requestSubtasks =
+    try {
+      LOG.debug("Search request: {}", postBody);
+
+      CurrentTraceContext currentTraceContext = Tracing.current().currentTraceContext();
+      try (var scope = new StructuredTaskScope<EsSearchResponse>()) {
+        List<StructuredTaskScope.Subtask<EsSearchResponse>> requestSubtasks =
           openSearchRequest.parseHttpPostBody(postBody).stream()
-              .map((request) -> scope.fork(currentTraceContext.wrap(() -> doSearch(request))))
-              .toList();
+            .map((request) -> scope.fork(currentTraceContext.wrap(() -> doSearch(request))))
+            .toList();
 
-      scope.join();
-      SearchResponseMetadata responseMetadata =
+        scope.join();
+        SearchResponseMetadata responseMetadata =
           new SearchResponseMetadata(
-              0,
-              requestSubtasks.stream().map(StructuredTaskScope.Subtask::get).toList(),
-              Map.of("traceId", getTraceId()));
-      return HttpResponse.of(
+            0,
+            requestSubtasks.stream().map(StructuredTaskScope.Subtask::get).toList(),
+            Map.of("traceId", getTraceId()));
+        return HttpResponse.of(
           HttpStatus.OK, MediaType.JSON_UTF_8, JsonUtil.writeAsString(responseMetadata));
+      }
+    } catch (Exception e) {
+      LOG.error("Error fulfilling request for multisearch query", e);
```

Comment:
I believe we should already be logging these, as we've enabled Armeria request logging here: https://github.com/slackhq/astra/blob/2d7c87f2e7d044474d2c3e051277bd485b5c66b6/astra/src/main/java/com/slack/astra/server/ArmeriaService.java#L72-L84

This should automatically log all successful and failed requests at the log levels indicated. This does make use of `responseContentSanitizer` which has since been deprecated in Armeria.

I'm not sure if this currently logs out the stack trace, but instead of adding try/catches to the individual endpoints I think that looking into the `LoggingService` decorator would be a better approach.